CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [Developer]: Updated metadata uploader to use react-router-dom Outlet. [See PR 1464](https://github.com/phac-nml/irida/pull/1464)
 * [Developer]: Deprecated "/api/projects/{projectId}/samples/bySequencerId/{seqeuncerId}" in favour of "/api/projects/{projectId}/samples/bySampleName", which accepts a json property "sampleName"
 * [Developer]: Fixed bug in setting a `default_sequencing_object and default_genome_assembly to `NULL` for a sample when the default sequencing object or genome assembly were removed. [See PR 1466](https://github.com/phac-nml/irida/pull/1466)
+* [Developer]: Fixed bug preventing a `sample` with and analysis submission from being deleted. [See PR 1467](https://github.com/phac-nml/irida/pull/1467)
```

Comment:
Minor typo here, can you replace "and" with "an", in the context of "with and analysis".

Reply:
Fixed in 9b91f37